build-gluster-org/jobs/smoke-template.yml

```diff
@@ -0,0 +1,76 @@
+- project:
+    name: 'smoke'
+    os_type:
+      - 32-bit-build:
+          node_name: fedora32
+          description: smoke job for 32 bit build - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/32-bit-build-smoke.sh
+          keyword: 32-bit-build 
+          timeout: 30
```

Comment:
Is there a way to get a default value for the timeout ? So we will add it only when it is different from the default, like blacklist_branch

Reply:
Yes, I will update it